src/puppetlabs/puppetdb/utils.clj

```diff
@@ -13,7 +13,8 @@
             [slingshot.slingshot :refer [try+ throw+]]
             [com.rpl.specter :as sp])
   (:import [java.net MalformedURLException URISyntaxException URL]
-           [org.postgresql.util PGobject]))
+           [org.postgresql.util PGobject]
+           puppetlabs.puppetdb.cheshire.RawJsonString))
```

Comment:
Now that I look closer, this isn't needed at all. That line can be removed and line 318 can be changed to (json/->RawJsonString payload)